### ISSUE TITLE
Why OpenSim?

### ISSUE THREAD

**Issue by thejoemarsh** [aclouthier/auto-marker-label#10]
Not an issue, just a question out of curiosity. 

What was the main reason for using OpenSim? If I'm understanding the paper correctly, couldn't you have used ezc3d with the clean datasets to calculate marker trajectories entirely inside the python scripts?

**Comment by aclouthier** [comment]
We used it to define marker sets to use to create the simulated marker trajectories for any new marker set. It was a convenient option to be able to visually place markers to match the marker set you used, and then we're using those positions on each segment with the inverse kinematics to generate simulated trajectories to use as training data for the algorithm. 
If you were only training the algorithm on existing data, then there isn't a need for OpenSim.

**Comment by thejoemarsh** [comment]
Ah I see, thanks for the info!
